keyboards/cannonkeys/devastatingTKL/bootloader_defs.h

```diff
@@ -0,0 +1,7 @@
+/* Address for jumping to bootloader on STM32 chips. */
```

Comment:
This file can be deleted as there is a default at `drivers/boards/GENERIC_STM32_F072XB/bootloader_defs.h` which will be used.